src/components/list/list.service.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const listModel = require('./list.model');
+const mongoose = require('mongoose');
+
+/**
+ * Post list
+ * @param {Object} list accountId, isDefault, name, isPublic & mediaItems
+ * @throws {Unauthorized} When credentials are wrong or not provided
```

Comment:
This is not true on this level. The method doesnt throw Unauthorized, the middleware takes care of that earlier on the request.